README.md

```diff
@@ -82,6 +82,16 @@ saucectl run
 
 This command will run the example test based on the `./.sauce/config.yml` file.
 
+
+### Using saucectl flags
+
+* Test timeout
```

Comment:
Maybe it's better to turn those into a level 4 header instead? Since the lines below the bullet point won't be indented anyway.
```suggestion
#### Test Timeout
```